src/components/Math/Math.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+
+import { operatorList } from '../../config/constants';
+
+const calculateResult = (first, second, operator) => {
+  if (operatorList.includes(operator)) {
+    // eslint-disable-next-line no-eval
+    const result = eval(`${first} ${operator} ${second}`);
+    return result;
+  }
```

Comment:
why we have to disable es lint ? can we directly return eval here instead of result ?